crates/bevy_asset/src/saver.rs

```diff
@@ -116,8 +116,17 @@ impl<'a, A: Asset> SavedAsset<'a, A> {
         Some(&labeled.asset)
     }
 
+    /// Returns the [`UntypedHandle`] of the labeled asset with the provided 'label', if it exists.
+    pub fn get_untyped_handle(
+        &self,
+        label: impl Into<CowArc<'static, str>>,
+    ) -> Option<&UntypedHandle> {
+        let labeled = self.labeled_assets.get(&label.into())?;
+        Some(&labeled.handle)
+    }
```

Comment:
```suggestion
    pub fn get_untyped_handle<Q>(
        &self,
        label: &Q,
    ) -> Option<&UntypedHandle> 
    where
        Cow<'static, str>: Borrow<Q>,
        Q: ?Sized + Hash + Eq,
    {
        let labeled = self.labeled_assets.get(label)?;
        Some(&labeled.handle)
    }
```
Using `Borrow` would avoid unnecessary conversion.

Alternatively, using `&str` for `label` or `AsRef<str>` should work too, but may become a bottleneck in some cases.

Reply:
Would require #11616 to be merged before

Reply:
11616 has been merged: let's use this approach.